### ISSUE TITLE
Posterior fat pad sign of the elbow

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10821]
**Preferred term label:**
Posterior fat pad sign of the elbow

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lateral elbow x-ray showing lucent crescent of fat that indicates a joint effusion in the elbow

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Supracondylar fracture
Radial head and neck fracture
Monteggia fracture
Olecranon fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001143
- label: Posterior fat pad sign of the elbow
- parents: [Abnormality of the elbow (HP:0009811)](https://hpo.jax.org/app/browse/term/HP:0009811)
- pmid: PMID:22735922
- def: Lateral elbow x-ray showing lucent crescent of fat that indicates a joint effusion in the elbow .
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10821
- synonyms:
